todo/app/controllers/tasks_controller.rb

```diff
@@ -34,7 +34,7 @@ def show
 
   # GET /tasks/
   def index
-    @tasks = Task.all
+    @tasks = Task.all.order('created_at DESC')
```

Comment:
Hashでも渡せます👍 
```suggestion
    @tasks = Task.all.order(created_at: :desc)
```
生SQL書かないといけない場面もあるけど、僕は極力ActiveRecordの機能に任せるようにしてます！

Reply:
cakephpの時生SQLにしたらうまく入力値エスケープしてくれなかった…とかもあったので、確かに極力ActiveRecordの機能使った方が良い気がします… 🤔 ありがとうございますー